### DISCUSSION TITLE
ChatGPT new conversation starter UX

### DISCUSSION THREAD

**Discussion by drnic** [AllYourBot/hostedgpt#564] [General]
Sharing the new UX in chatgpt for conversation starters.

1. Some one word buttons beneath text input
    <img width="872" alt="Screenshot 2024-11-25 at 9 11 10 am" src="https://github.com/user-attachments/assets/3f1b5e41-bce5-4b7e-9a71-85d3b4101f5e">
2. Becomes a prefix in the text area and a new autocomplete dropdown.
    <img width="981" alt="Screenshot 2024-11-25 at 9 11 15 am" src="https://github.com/user-attachments/assets/d0ba87c5-c97b-4d4b-82be-f876d9b2df98">
3. WHich becomes an even more rich initial user message prompt.
    <img width="1005" alt="Screenshot 2024-11-25 at 9 11 26 am" src="https://github.com/user-attachments/assets/2f456c85-89ff-4872-9252-4ffc7a5b10d4">

**Comment by keithschacht** [comment]
That's cool. But I've never found their starters / suggested prompts very helpful. But in general, I really like the UI updates all around. The text input area is improved, the sidebar, and the chat bubbles too. The current HostedGPT UI was a near pixel perfect match for what ChatGPT has done, but I haven't had time to re-implement this new UI. I've been hoping someone comes along who is great with CSS who can help me.
